### DISCUSSION TITLE
blog/10-potential-career-options-with-a-degree-in-statistics/

### DISCUSSION THREAD

**Discussion by giscus** [AntoineSoetewey/statsandr#105] [General]
# blog/10-potential-career-options-with-a-degree-in-statistics/

In this post, I present 10 potential career options available with a degree in statistics, with a description and the required skills to pursue these jobs

https://statsandr.com/blog/10-potential-career-options-with-a-degree-in-statistics/

**Comment by hansfriese** [comment]
Thank you, it's helpful.

**Reply by hansfriese** [reply]
@hansfriese Thanks

**Reply by AntoineSoetewey** [reply]
You're welcome @hansfriese!

**Comment by irenerlover** [comment]
Desr Antoine, 
I am very interested in your PhD experience. You mentioned that "My research focuses on applying biostatistical procedures to cancer patients, so it is a rather applied PhD, but many of my colleagues work on a more theoretical subject."

Did it mean that you are an applied statistician? What is the difference between applied statistician and statistician? (I know I am asking a silly question but I am curious about your PhD program.) Also, would you mind to share the courses you registered during PhD? I hope that I can do more self study.

Thank you for your time. Sorry for my poor English. I am glad to find someone who is willing to share his/her experience. Appreciate your efforts!!!!

Best regards,
Irene

**Reply by AntoineSoetewey** [reply]
Dear Irene,

> Did it mean that you are an applied statistician?

Yes.

> What is the difference between applied statistician and statistician?

I don't think there is a formal definition, but you can consider that an applied statistician is more likely to apply statistical techniques and methods (which are already known) on some real data, usually to answer a practical question or issue, rather than developing new statistical tools. Of course, nothing prevents an applied statistician to develop innovative techniques and a more theoretical statistician to analyze real data: it is all about what she prefers to work on. And it can even change throughout her career.

> Also, would you mind to share the courses you registered during PhD?

I don't mind sharing the courses I followed. Just bear in mind that it can be very different from one person to another because it usually depends on the courses you already followed, your university, your preferences, the field of your PhD thesis (mine is in survival analysis, so I followed courses in this field). That being said, here are the courses I followed:

- Analysis of survival and duration data
- Analysis of net survival and excess mortality: concepts and methods
- Clinical trial statistics
- Linear mixed-effects models with R
- and a couple of workshops held during conferences

This in addition to [the courses I teach](https://antoinesoetewey.com/teaching/) as part of my teaching tasks.

> I hope that I can do more self study.

There are often workshops or training organized by the university. I believe they are a great starting point in addition to your study program in case your are interested in deepening your knowledge in statistics and programming.

Hope this helps.

Regards,
Antoine

**Reply by irenerlover** [reply]
I have already graduated from the university so I can only do self study through internet (like your blog!) or join some workshop... Anyway, thanks for your suggestion and reply^^ One more question, is it necessary to study real analysis before enrolling the PhD program? I am trying to read some book about measure theory and think that it will be more effective if I can have a lecture instead of self study...

**Reply by AntoineSoetewey** [reply]
> One more question, is it necessary to study real analysis before enrolling the PhD program?

I cannot say it is *necessary* because it depends on the requirements of your PhD program, your grades, whether the program you are applying to is more theoretical or more applied, etc. If it is more applied, it is undeniable that showing examples of analyses you have done in the past (if possible, in the field you are applying) will be an asset for your application.

**Comment by joshuamarie** [comment]
Most of the jobs are currently saturated...f******k! (I am sorry for being like this). The most possible and fitted jobs to me are:

-  (Freelance) Consultant
-  PhD student

However, for PhD student, until I completed my R package project called [{kindling}](https://github.com/joshuamarie/kindling), I have no plan to take one just yet.
